frontend/javascripts/admin/tasktype/task_type_create_view.js

```diff
@@ -226,131 +225,144 @@ class TaskTypeCreateView extends React.PureComponent<Props, State> {
                 </span>
               }
               hasFeedback
+              rules={[{ required: true }]}
             >
-              {getFieldDecorator("description", {
-                rules: [{ required: true }],
-              })(<TextArea rows={10} />)}
+              <TextArea rows={10} />
             </FormItem>
 
-            <FormItem label="Annotation Type">
-              {getFieldDecorator("tracingType", {
-                initialValue: "skeleton",
-              })(
-                <RadioGroup>
-                  <Radio value="skeleton" disabled={isEditingMode}>
-                    Skeleton
-                  </Radio>
-                  <Radio value="volume" disabled={isEditingMode}>
-                    Volume
-                  </Radio>
-                  <Radio value="hybrid" disabled={isEditingMode}>
-                    Skeleton and Volume
-                  </Radio>
-                </RadioGroup>,
-              )}
+            <FormItem
+              name="tracingType"
+              label="Annotation Type"
+              onChange={() => this.forceUpdate()}
```

Comment:
Looks to me like this was added. Why is this needed?

Reply:
@daniel-wer The blame shows, that you added this line.

Reply:
Initially, I added this because a change in one antd form item no longer would trigger a rerender of the whole form with the new version and part of the form actually needs to be rerendered if this item is changed. However, I solved this more elegantly later, using the `shouldUpdate` prop of form items. I just tested that this line is no longer needed and will remove it. Thank you for paying close attention :)